src/utils.jl

```diff
@@ -229,4 +233,25 @@ function generate_forces_from_calculator(calc_type)
         $q1
         $q2
     end
+end
+
+
+function generate_calculator_hessian(calc_type)
+    q = quote 
+        function AtomsCalculators.calculate(::AtomsCalculators.Hessian, system, calculator::$calc_type; kwargs...)
+            h = AtomsCalculators.hessian(system, calculator; kwargs...)
+            return ( hessian = h, )
+        end
+    end
+    return q
+end
+
+function generate_hessian(calc_type)
+    q = quote
+        function AtomsCalculators.hessian(system, calculator::$calc_type; kwargs...)
+            h = AtomsCalculators.calculate(AtomsCalculators.Hessian(), system, calculator; kwargs...)
+            return h[:hessian]
+        end
+    end
+    return q
 end
```

Comment:
There is a lot of code duplication in this file for the generates. Can this not be simplified somehow by an `@eval` inside a `for` loop or similar ?

Reply:
(Just a suggestion, no need to do that now)

Reply:
Yes, I noticed that too. Although this was not an issue untill now, so it was just faster to add these two.